packages/flame/lib/src/extensions/vector2.dart

```diff
@@ -50,6 +50,12 @@ extension Vector2Extension on Vector2 {
     }
   }
 
+  /// Scale this vector by the given [scale].
+  void scaleBy(Vector2 scale) {
```

Comment:
This already exists in the package?

Reply:
this is just simple multiplication right?

Reply:
Mmm it is indeed, I think I added this when the multiplication wasnt working... But in hindsight that might be because I was using an `Offset`...